presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -244,6 +244,44 @@ Approximate Aggregate Functions
     Each element of the array must be between zero and one, and the array must
     be constant for all input rows.
 
+.. function:: approx_percentile(x, w, percentages, accuracy) -> array<[same as x]>
```

Comment:
Would you extract the addition of this function into a separate commit?

Reply:
Sure. would you prefer it to be a separate PR too?

Reply:
No need for separate PR. Just separate commit.